README.md

```diff
@@ -213,6 +213,15 @@ Optionally, you can also pass the template directory to which you want to instal
 $ sh -c "$(curl -fsSL https://r.viktoradam.net/githooks)" -- --use-core-hookspath --template-dir /home/public/.githooks
 ```
 
+When you want to install from another url (e.g. from your own fork) you can specify the release repository url 
+as well as the branch name (default: `master`) when installing with:
+
+```shell
+sh -c "$(curl -fsSL https://r.viktoradam.net/githooks)" -- --release-clone-url "https://server.com/my-githooks-fork.git" --release-clone-url "release"
+```
+
+which will then be used for installation and further updates.
```

Comment:
```suggestion
If you want to install from another repository (e.g. from your own fork), you can specify the release repository URL 
as well as the branch name (default: `master`) when installing with:

```shell
sh -c "$(curl -fsSL https://r.viktoradam.net/githooks)" -- --release-clone-url "https://server.com/my-githooks-fork.git" --release-clone-branch "release"
```

This will be then used for installation and further updates.
```